spec/lib/traffic_source_parser/parser/utmz_parser_spec.rb

```diff
@@ -36,7 +36,7 @@
           let(:campaign) { 'experimentar-ferramenta' }
           let(:medium) { 'sponsored-post' }
           let(:channel) { 'Social' }
-          let(:source) { 'facebook' }
+          let(:source) { 'Facebook' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.